Documentation/common-problems.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Common Problems
+weight: 42
+indent: true
+---
+
+# Common Problems and Troubleshooting
+
+Many of these problem cases are hard to summarize down to a short phrase that adequately describes the problem. Each problem will start with a bulleted list of symptoms. Keep in mind that all symptoms may not apply depending upon the configuration of the cluster and/or Rook.io. If the majority of the symptoms are seen there is a fair chance you are experiencing that problem.
+
+If after trying the suggestions found on this page and the problem is not resolved, the Rook.io team is very happy to help you troubleshoot the issues in their Slack channel. Once you have [registered for the Rook.io Slack](https://rook-slackin.herokuapp.com/), proceed to the General channel to ask for assistance.
+
+- [Cluster failing to service requests](#cluster-failing-to-service-requests)
+- [Reinstall fails to start all Pods](#reinstall-fails-to-start-all-pods)
+
+
+
+## Cluster failing to service requests
+
+* Execution of the `ceph` command hangs
+* Execution of the `rookctl` command hangs
+* PersistentVolumes are not being created
+* Large amount of slow requests are blocking
+* Large amount of stuck requests are blocking
+* One or more MONs are restarting periodically
+
+### Further Details
+Create a rook-tools pod to investgate the current state of CEPH. Here is an example of what one might see. In this case the `ceph status` command would just hang so a CTRL-C needed to be send. The `rookctl status` command is able to give a good amount of detail. In some cases the rook-api pod needs to be restarted for `rookctl` to be able to gather information. If the rook-api is restarted, then the rook-tools pod should be restarted also. 
```

Comment:
- You might call this section `Investigation`, then split up this paragraph into bullet points of things to try. 
- Restarting the rook-api pod shouldn't be necessary anymore since #990 was merged